runtime/experimental/Makefile

```diff
@@ -1,46 +1,23 @@
 SHELL := /bin/bash
 
-REGION ?= us-east-2
-BREF_VERSION ?= 0.3.0
-PHP_VERSION_SHORT ?= 7.2
-S3_BUCKET ?= bref-php-${REGION}
-PHP_VERSION_SHORT_UNDERSCORE = $(subst .,_,${PHP_VERSION_SHORT})
-
 php: build publish
 
+# Build the docker container that will be used to compile PHP and its extensions
 compiler:
 	docker build -f ${PWD}/compiler.Dockerfile -t bref/runtime/compiler:latest .
 
+# Compile PHP and its extensions
 build: compiler
 	docker build -f ${PWD}/php.Dockerfile  -t bref/runtime/php:latest $(shell helpers/docker_args.php php72) .
 
+# Export the compiled PHP artifacts into zip files that can be uploaded as Lambda layers
 distribution:
 	rm -rf export/
 	docker build --no-cache -f ${PWD}/export.Dockerfile -t bref/runtime/dist:latest .
 	docker run --name bref-export-container bref/runtime/dist:latest /bin/true
 	docker cp  bref-export-container:/export .
 	docker rm  bref-export-container
 
+# Publish the layers on AWS Lambda
 publish: distribution
-	$(eval LAYER_NAME = bref-php-default-${PHP_VERSION_SHORT_UNDERSCORE})
-	$(eval S3_KEY = ${BREF_VERSION}/php-default/${PHP_VERSION_SHORT}/php.zip)
-
-	# Upload to S3
-	#aws s3 cp export/php-${PHP_VERSION_SHORT}-cli.zip s3://${S3_BUCKET}/${S3_KEY}
-	#sleep 15
-
-	# Publish the layer
-	$(eval LAYER_VERSION = $(shell aws lambda publish-layer-version --region=${REGION} --layer-name ${LAYER_NAME} --description "bref PHP ${PHP_VERSION} Default" --license-info "MIT" --zip-file fileb://export/php-${PHP_VERSION_SHORT}-cli.zip --compatible-runtimes provided --output text --query Version))
```

Comment:
Kept the `--zip-file` option (in the new script) which avoids going through S3 🎉

Reply:
I think that is a good idea. I also noticed that seemed a lot faster on uploads. But maybe that was just wishful thinking. :-)